### ISSUE TITLE
labels for vggface2 model

### ISSUE THREAD

**Issue by yehiyam** [timesler/facenet-pytorch#60]
Hi 
when I do classification with the pretrained  vggface2 model, how do I match a probability to a name? do you have a list of labels for the trained model?

Thanks

**Comment by Gilf641** [comment]
Are you using Tensorflow? If yes then I do have a solution...

**Comment by f0ti** [comment]
The list doesn't seem right. At least it looks different from the list provided in the Identity Meta_v1. file on VGGFACE2 website : https://www.robots.ox.ac.uk/~vgg/data/vgg_face2/meta_infor.html.

**Comment by Jiaqi0602** [comment]
Hi, can you provide the folder structure for VGGFACE2 datasets when training? I'm using the vggface2 pretrained model to make evaluation on the data but the label of the prediction seems inaccurate. Thanks.

**Comment by alem-memic** [comment]
I have a problem because in the paper https://www.robots.ox.ac.uk/~vgg/publications/2018/Cao18/cao18.pdf the number of labels is 9131, but the output shape with probabilities is with dimension 8631. Why is this the case?

**Comment by jfthuong** [comment]
After doing the fine tuning as explained in  https://github.com/timesler/facenet-pytorch/blob/master/examples/finetune.ipynb  , I am doing the following code to create a dictionary of probability for each person:

```python
from PIL import Image
path = "<path to my picture>"

mtcnn = MTCNN(
    image_size=160, margin=0, min_face_size=64,
    thresholds=[0.6, 0.7, 0.7], factor=0.709, post_process=True,
    device=device
)

img = mtcnn(Image.open(path), save_path=None)
probs = resnet(img.unsqueeze(0).to(device))

{name:prob for name, prob in zip(dataset.classes, list(probs[0].tolist()))} 
```

**Comment by MisdakM** [comment]
> I have a problem because in the paper https://www.robots.ox.ac.uk/~vgg/publications/2018/Cao18/cao18.pdf the number of labels is 9131, but the output shape with probabilities is with dimension 8631. Why is this the case?

As stated in the [repo description](https://github.com/timesler/facenet-pytorch/tree/master#:~:text=For%20VGGFace2%2C%20the%20pretrained%20model%20will%20output%20logit%20vectors%20of%20length%208631%2C%20and%20for%20CASIA%2DWebface%20logit%20vectors%20of%20length%2010575), the model output an embedding vector of 8631 logits which represents the number of identities in the training set of VGGFace2,

The VGGFace2 dataset is made of around 3.31 million images divided into 9131 classes, each representing a different person identity. The dataset is divided into two splits, one for the training and one for test. The latter contains around 170000 images divided into 500 identities while all the other images belong to the remaining **8631** classes available for training.

**Comment by AnubhavBharadwaaj** [comment]
Can someone provide me download link for VGGFace2-pretrained ResNet-50 . The official link has stopped working https://www.robots.ox.ac.uk/~vgg/data/vgg_face2/models/ . Please. 
Thanks!
